sys/src/main/java/com/alibaba/nacos/sys/utils/InetUtils.java

```diff
@@ -159,8 +160,9 @@ public static InetAddress findFirstNonLoopbackAddress() {
                     if (!ignoreInterface(ifc.getDisplayName())) {
                         for (Enumeration<InetAddress> addrs = ifc.getInetAddresses(); addrs.hasMoreElements(); ) {
                             InetAddress address = addrs.nextElement();
-                            if (address instanceof Inet4Address && !address.isLoopbackAddress() && isPreferredAddress(
-                                    address)) {
+                            if ((IPUtil.PREFER_IPV6_ADDRESSES ? address instanceof Inet6Address
+                                    : address instanceof Inet4Address) && !address.isLoopbackAddress()
+                                    && isPreferredAddress(address)) {
```

Comment:
`IPUtil.PREFER_IPV6_ADDRESSES ? address instanceof Inet6Address : address instanceof Inet4Address`  
Here is a separate line, which is more readable.